bin/cheribsdtest/cheribsdtest_registers.c

```diff
@@ -214,8 +214,10 @@ check_initreg_code(void * __capability c)
 	if ((v & CHERI_PERM_SEAL) != 0)
 		cheribsdtest_failure_errx("perms %jx (seal present)", v);
 
+#ifdef CHERI_PERM_CCALL
```

Comment:
We should probably be defining CHERI_PERM_CCALL as an alias for BRANCH_SEALED_PAIR and checking it is present

Reply:
I tried that and test_initregs_stack and test_initregs_pcc fail.

Reply:
Oh, that's because arm64's cherireg.h doesn't include CHERI_PERM_BRANCH_UNSEAL in CHERI_PERMS_USERSPACE. That's wrong.

Reply:
(and it should really call that BRANCH_SEALED_PAIR to match the name in the specification)

Reply:
Should all be fixed on morello-dev with a few commits you can cherry-pick (dropping the cheribsd changes that just revert things back to how they are on dev).

Reply:
Thanks!